src/features/projectsV2/ProjectsMap/SingleProjectView.tsx

```diff
@@ -36,12 +41,15 @@ const SingleProjectView = ({ mapRef, setIsOnSampleMarker }: Props) => {
       features: singleProject?.sites ?? [],
     };
   }, [projectSlug]);
-  // Zoom to plant location polygon
+  // Zoom to plant location
   useEffect(() => {
-    const isPlantLocationReadyToZoom =
-      selectedPlantLocation && router.isReady && requestedPlantLocation;
+    const canZoomToPolygonPlantLocation =
+      selectedPlantLocation &&
+      router.isReady &&
+      requestedPlantLocation &&
+      !selectedSamplePlantLocation;
```

Comment:
Make it a point to ensure that a variable with a boolean name contains a boolean value, instead of depending on implicit truthyness/falsyness.